pkg/cmd/attestation/verification/mock_verifier.go

```diff
@@ -44,12 +50,51 @@ func (v *MockSigstoreVerifier) Verify(attestations []*api.Attestation, policy ve
 	return results, nil
 }
 
-func NewMockSigstoreVerifier(t *testing.T) *MockSigstoreVerifier {
-	return &MockSigstoreVerifier{t}
+func NewMockSigstoreVerifier(t *testing.T, mockResults []*AttestationProcessingResult) *MockSigstoreVerifier {
+	return &MockSigstoreVerifier{t, mockResults}
+}
+
+func NewDefaultMockSigstoreVerifier(t *testing.T) *MockSigstoreVerifier {
```

Comment:
food for thought / not high priority suggestion: considering the nature of this change i'd invert this: `NewMockSigstoreVerifierWithMockResults` so we don't modify every other usage of it / i feel like it's harder to reason what the Default Mock is.

Reply:
Agreed, we can definitely iterate on this one